src/gui/screen_menu_lan_settings.cpp

```diff
@@ -109,13 +111,31 @@ ScreenMenuLanSettings::ScreenMenuLanSettings()
     msg_shown = false;
 }
 
+uint32_t ScreenMenuLanSettings::StringifyNetworkDevice(uint32_t netdev_id, lan_descp_str_t buffer) {
```

Comment:
Style consistency: Methods here seem to start with lower case letter, upper case is for types/clases.

What does the return type mean and is it in any way useful? The only caller below doesn't use it, shall it be `void` then? Besides, it is wrong ‒ `snprintf` returns `int`.